src/lib.rs

```diff
@@ -84,3 +106,44 @@ fn makecircle(centerpoint: Point<f64>, radius: f64, num_vertices: usize) -> Poly
     let polygon = Polygon::new(LineString::from(circle_points), vec![]);
     polygon
 }
+
+// Make a single clock polygon
+pub fn clockpoly(
+    centerpoint: Point<f64>,
+    radius: f64,
+    inner_radius: f64,
+    num_vertices: usize,
+    num_segments: usize,
+    seg: usize,
+) -> Polygon<f64> {
+    let mut circle_points = Vec::new();
+    let mut circle_points_inner = Vec::new();
+
+    // Sequence of vertices
+    // in R round(seq(1, 13, length.out = 12))
+    // Number of vertices per segment
+    let n = num_vertices / num_segments;
```

Comment:
![Screenshot from 2021-04-01 09-53-00](https://user-images.githubusercontent.com/1664407/113327649-0f270380-92d0-11eb-891c-73b746186ae6.png)
The gaps are coming from this integer division, I think.

Reply:
This is fixed with the following:

```
    let n = (num_vertices / num_segments) + 1;
    let f = seg * n;
    let t = 1 + (seg + 1) * n;
```